### PR TITLE
Add 3 phishing URL

### PR DESCRIPTION
**Pull request by bulkagaint**

### CODE REVIEW

File: add-link [hunk 1]
```diff
@@ -38,4 +38,6 @@ https://www.mdscbenin.org/wp-includes/theme-compat/dar/newsant1/login.php
 https://nort2on.com/setup/
 http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
-
+https://copyrihgts-infringement.tk/contact/192193817099/quadrojoseph.txt
+https://copyrihgts-infringement.tk/contact/192193817099/
+http://ighelpmail.com
```
Comment (spirillen) [inline add-link:43]: image

What/Where is the Phishing or scam?

**Review by spirillen — CHANGES_REQUESTED** [review]

File: add-link (hunk 1 above)
Comment (bulkagaint) [reply add-link:43]: Webmail is installed on this domain and fraudulent mails are sent from here. I'm forwarding the mail header

```
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=mime-version:cc:message-id:subject:date:to:from;
        bh=14sHU3QFAWRwROlhrHbQxUdcTdBmb1ioSAmiA5/wtfo=;
        b=fAug4qjLhbTEP3JPede6EohqrWis9GFGxLSe1uEOcMMH+ndsJR3GzTtPdAgKsUULTH
         Izs8jqqPa7AFlKLArUVXOUmbcAHhbz4twvCOXVoHBk75It/lqSSoKB5ZPcS0/lo29vC4
         v8tN8OZS/qMpA0pRMIvvLPBTRGs0MG16BcCVM+hYLE1Kf+H7RxA/zfSujOFems2iRufp
         lIaMNRpGu+ackQBRHp/6YbIqHknKTunmdrnvZbDsYHSJ6POM8egzrPULIjJq2wcGt7HB
         qD86DzXzZwLmLiXQuv506ggXGRQDfJMDKA2rE7RuPHjfxWa8bWn4TnOyB7p51n08GyT5
         RyuQ==
ARC-Authentication-Results: i=1; mx.google.com;
       spf=pass (google.com: domain of mail@ighelpmail.com designates 216.87.78.66 as permitted sender) smtp.mailfrom=mail@ighelpmail.com
Return-Path: <mail@ighelpmail.com>
Received: from smtpout1.namebrightmail.com (smtpout1.namebrightmail.com. [216.87.78.66])
        by mx.google.com with ESMTPS id p20si9859094ioo.40.2021.08.06.12.47.30
        for 
        (version=TLS1 cipher=ECDHE-ECDSA-AES128-SHA bits=128/128);
        Fri, 06 Aug 2021 12:47:30 -0700 (PDT)
Received-SPF: pass (google.com: domain of mail@ighelpmail.com designates 216.87.78.66 as permitted sender) client-ip=216.87.78.66;
Authentication-Results: mx.google.com;
       spf=pass (google.com: domain of mail@ighelpmail.com designates 216.87.78.66 as permitted sender) smtp.mailfrom=mail@ighelpmail.com
Received: by smtpout1.namebrightmail.com with smtp id nb_16667275;
 	 Fri, 06 Aug 2021 13:47:30 -0600
Received: from www.namebrightmail.com [216.150.213.194] by www.namebrightmail.com with smtp (ENV-FROM: mail@ighelpmail.com);
	 Fri, 06 Aug 2021 13:47:25 -0600
From: mail@ighelpmail.com
To:  mymail 
Date: Fri, 06 Aug 2021 13:47:24 -0600
Subject: 
Message-Id: <C30RUEZZFEU4.I5K4DKZ2DZ5X1@TurnCommerce50>
```

Comment (spirillen) [reply add-link:43]: Just the issue that the scam mails are sent from

 `66.78.87.216.in-addr.arpa.      3600    IN      PTR     smtpout1.namebrightmail.com.`

and not

`ighelpmail.com. 300     IN      A       3.209.52.151`

What would be needed to confirm this is the raw output of the mail contents itself...

```
drill ighelpmail.com @9.9.9.9 -t mx
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 24417
;; flags: qr rd ra ; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; ighelpmail.com.      IN      MX

;; ANSWER SECTION:
ighelpmail.com. 10800   IN      MX      10 mail2.namebrightmail.com.
ighelpmail.com. 10800   IN      MX      10 mail1.namebrightmail.com.
```

Comment (spirillen) [reply add-link:43]: This said, there are some other notes for the domain `ighelpmail.com` on https://github.com/referefref

  - https://jamesbrine.com.au/phishingdomains-2021-02-06/
  - https://jamesbrine.com.au/phishingdomains-2021-02-08/

**Pull request closed without merging**